tensorflow/compiler/xla/service/gpu/alpa_nccl_wrapper.h

```diff
@@ -44,55 +44,77 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
+Status ncclResultToStatus(ncclResult_t s, const char* file, int64_t line,
+                          const char* expr);
+
+#define XLA_CUDA_STATUS(expr) \
+  xla::gpu::ncclResultToStatus(expr, __FILE__, __LINE__, #expr)
+
+#define XLA_CUDA_RETURN_IF_ERROR(expr) \
+  do {                                 \
+    Status s = XLA_CUDA_STATUS(expr);  \
+    if (!s.ok()) {                     \
+      return s;                        \
+    }                                  \
+  } while (0)
+
+#define XLA_CUDA_WARN_IF_ERROR(expr)  \
```

Comment:
Copy the code is not good. Later we should try to resolve the build problem

Reply:
Yes, it is indeed because I could not import the code from nccl_utils.cc .